Automattic/wc-calypso-bridge
# Add filter for recommended WPCOM themes

### Changes proposed in this Pull Request:

Adds a filter to change the recommended themes for Woo Express sites on WPCOM. 

Closes #1316.



### How to test the changes in this Pull Request:




1. Follow the instructions to setup a Woo Express site to test CYS: pdibGW-2pG-p2
2. Update the `wc-calypso-bridge` plugin with this branch.
3. Visit `My Home` in `wp-admin` and click "Customize Your Store"
4. The Intro page will show 4 themes; confirm that the site URL is appended to the theme links, eg: `https://wordpress.com/theme/tazza/yourblog.com`
5. Activate one of the recommended themes (Tsubaki, Tazza, Amulet, or Zaino) via the Themes showcase
6. Re-visit the "Customize Your Store" page and confirm that active theme is marked as such:
<img width="409" alt="Screen Shot 2023-10-17 at 2 57 39 PM" src="https://github.com/Automattic/wc-calypso-bridge/assets/4297811/a3e035d7-e323-4b44-a4f7-55174055d830">



### Other information:




### FOR PR REVIEWER ONLY:
